### DISCUSSION TITLE
Please focus on a stable version, and a beta channel for new developments

### DISCUSSION THREAD

**Discussion by erikgeurts** [Clooos/Bubble-Card#325] [Questions about config, custom styles and templates]
While your project shows great promise, I've been put off by the constant stream of bug fixes and frequent releases. This also affects the reputation of the project as a whole, in my opinion.

I'd like to recommend that you focus on two things:
- a stable version that doesn't change too frequently (at most once or twice per month, perhaps even less)
- a beta version for new developments and experiments, that people that have a more daring attitude can try out

Once a new development has all the kinks ironed out in the beta channel, and has been without issues for a reasonable period of time, it can be promoted to the stable version for regular users.

I hope you will consider my suggestion.

**Comment by Clooos** [comment]
Hi and thanks for your feedback! I can't agree more, 1.5.0 was a nightmare even with a beta. But I rushed a bit these stable releases to fix the huge performance issues.

It was probably a mistake, I'm still new to all of this but I'm not planning to add more breaking changes in the future.

I will also release more beta versions 👍
